### DISCUSSION TITLE
I need help from an Obi Won

### DISCUSSION THREAD

**Discussion by Scoopdiggity** [sympa-community/sympa#1799] [General]
Hello,
I work for a stunningly large company that has me using SYMPA to send out 100k-400k email per month, sometimes more.  And while we're not throwing errors, there's always massive issues with consistency.  Our head admin has no manuals, and i'm using a GUI as the front end.  I wouldn't know how to handle terminal commands anyway.  I'm a marketer with some technical skills - but not to the point where I can wrestle this beast. Is anyone aware of a group that does training classes? Like if i needed help with JIRA or CONFLUENCE - I'd be able to find that.  I would travel for a class if one could be found.  I've been using it a long time - and there's still no user guide.  Any help would be appreciated.  I'm listing some issues below:

- No recorded bounces even though I sent to 158 email addresses that did not exist
- Sometimes it asks for moderation...sometimes it doesn't.
- Sometimes, if i paste the same name twice, I will get a message like "XXXX is already a subscriber," but sometimes, it will send a dozen copies to ONE or TWO people out of 75 thousand.

Any help would be appreciated.
Thanks,
Seth

**Comment by ikedas** [comment]
Unfortunately, I think there is little we here can do to help.

Looking at the issues you have listed, I think it is possible that there are some problems with the mail delivery system and not simply with Sympa.

It is the job of the system maintenance team to investigate and resolve such issues from a comprehensive perspective. You should ask them to investigate.
